tests/playwright/src/model/pages/details-page.ts

```diff
@@ -58,4 +60,25 @@ export abstract class DetailsPage extends BasePage {
       return this;
     });
   }
+
+  async executeCommandInTerminal(command: string): Promise<void> {
+    await playExpect(this.terminal).toBeVisible();
```

Comment:
You can add terminal tab activation into this function as it cannot be performed without it.